stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/MessageUtils.kt

```diff
@@ -30,6 +30,11 @@ internal fun Message.isRegular(): Boolean = type == ModelType.message_regular
  */
 internal fun Message.isSystem(): Boolean = type == ModelType.message_system
 
+/**
+ * @return If the message type is error.
+ */
+public fun Message.isError(): Boolean = type == ModelType.message_error
```

Comment:
Do we need to make this public?

Reply:
I would either make it internal, like `isSystem()` or make both of them public!

Reply:
Made it internal